version-check

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env bash
+set -eu
+
+library="java"
+weblog_variant="ratpack"
+
+readonly OUTPUT_DIR=version-check-output
+mkdir -p "$OUTPUT_DIR"
+
+declare -A SEEN_XPASS
+declare -A SEEN_FAIL
+
+versions=(
+    0.110.0
```

Comment:
is there a way to get this lit automatically ?

Reply:
I'm guessing yes, but it would have to be a different method for each library, so it's a bit annoying to implement in a single script like that. What if we accepted the list from stdin instead ?

Reply:
> I'm guessing yes, but it would have to be a different method for each library, so it's a bit annoying to implement in a single script like that

it wouldn't be the first time 😄 

> What if we accepted the list from stdin instead ?

That's another possibility (actually compatible with the first solution).

But in a sense, I feel that the hardcoded solution is not the good one : maintenance, and possibly missing version. Actually, I prefer : 

1. @simon-id's  idea (list provided in stdin)
2. nice to have : a script that load that list (giving the liberty to each tracer team to implement it if they feel it's needed).